controllers/auth.js

```diff
@@ -39,4 +65,51 @@ exports.signUp = async ( req, res, next) => {
     } catch (e) {
         next(e)
     }
+}
+
+exports.signIn = async (req, res, next) => {
+    try {
+        const email = req.body.email
+        const password = req.body.password
+        
+        const loadedUser = await User.findOne({email: email})
+        
+        if(!loadedUser) {
+            const error = new Error('user with that email not exists')
+            error.statusCode = 404
+            throw error
+        }
+        
+        const isEqual = bcrypt.compare(password, loadedUser.password)
+            
+        if(!isEqual) {
+            const error = new Error('wrong password')
+            error.statusCode = 401
+            throw error
+        }
+            
+        const token = jwt.sign({
+            email: loadedUser.email,
+            userId: loadedUser._id.toString(),
+            role: loadedUser.role
+        }, 'uniwersytetkazimierzawielkiegowbydgoszczy', {
```

Comment:
Please add this key(`uniwersytetkazimierzawielkiegowbydgoszczy`) as an environment variable.

Reply:
PLIZ SRIZ